arcade/camera/camera_2d.py

```diff
@@ -434,6 +434,12 @@ def center_left(self, new_left: Point2):
         x, y = new_left
         self.position = x - up[1] * left, y - up[0] * left
 
+    def position_rect(self) -> Rect:
+        """ Get the LRBT rect of the camera position """
+        bx, by = self.bottom_left
+        tx, ty = self.top_right
+        return LRBT(left=bx, right=tx, bottom=by, top=ty)
```

Comment:
Because the Camera2D can rotate, it doesn't make sense to return a rect as you have defined it. However, I think this is a valuable feature.

```python
def aabb(self) -> Rect:
    """
    Retrieve the axis-aligned bounds box of the camera's view area. 
    If the camera isn't rotated , this will be precisely the view area, 
    but it will cover a larger area when it is rotated.
    """
    tr_x, tr_y = self.top_right
    tl_x,, tl_y = self.top_left
    br_x, br_y = self.bottom_right
    bl_x, bl_y = self.bottom_left
    left = min(tl_x, tr_x, bl_x, br_x)
    right = max(tl_x, tr_x, bl_x, br_x)
    bottom = min(tl_y, tr_y, bl_y, br_y)
    top = max(tl_y, tr_y, bl_y, br_y)
    return LRBT(left=left, right=right, bottom=bottom, top=top)
```

Reply:
True! Changed as you suggested